### ISSUE TITLE
First command to neovim doesn't work

### ISSUE THREAD

**Issue by rgrinberg** [neovim/python-client#83]
The example taken from here that is: http://neovim.org/doc/user/msgpack_rpc.html#msgpack-rpc

Steps to reproduce:

1) Turn on neovim

2) Connect with python

Here's what I'm running ipython:

```
In [1]: from neovim import attach
In [2]: nvim = attach('socket', path='/var/folders/1y/41fddxzj4sbdkwxlq_y92zsw0000gn/T/nvimEE9Hyn/0')
In [3]: nvim.command('echo "testing"')
```

3) Now the last command blocks indefinitely.

4) Go to neovim

5) Press key and the python command will unblock

6) Subsequent commands sent with python work and execute immediately.

**Comment by equalsraf** [comment]
Hi, I am unable to replicate this on my end, can you provide some more details?

- which revision of Neovim are you using (and where did you get it homebrew/etc)
- do other commands work? e.g. `nvim.current.line`
- just to be sure can you test using nvim -u NONE

Sidenote: vim_command is a deferred command, maybe it is related to that, but that command should never block. There was similar issue earlier due to a bug in libuv - but it only manifested in Windows.
